src/mixins/filing-template-mixin.ts

```diff
@@ -365,21 +365,12 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           legalType: this.getEntityType,
           nrNumber: this.getNameRequest.nrNumber
         },
-        parties: this.getAddPeopleAndRoleStep.orgPeople,
+        parties: this.orgPersonsToParties(this.getAddPeopleAndRoleStep.orgPeople),
```

Comment:
The type conversion is done here, as close as possible to the interface between the UI and the API (since the UI calls it one thing and the API calls it another).

Ditto for line 438 below.